### DISCUSSION TITLE
Some symbols are not rendering properly in bash

### DISCUSSION THREAD

**Discussion by gutsytechster** [starship/starship#1248] [General]
## Bug Report
I just installed `starship` on my bash. However, when I moved to a Python repository, I see that the `snake` symbol doesn't appear properly.

#### Current Behavior

Currently, the prompt appears as

![starship_error](https://user-images.githubusercontent.com/31796326/82748853-b7239480-9dc2-11ea-9e8a-4cf8ab1372de.png)

#### Expected Behavior

It should be appeared as 

`pip on  master via 🐍 v3.7.4 via C base `

#### Additional context/Screenshots

I experience this problem with some other symbols as well.

#### Possible Solution

I know that this must be an issue with the shell. But I don't know exactly what to look for, for proper rendering or what settings should I set.

#### Environment
- Starship version: starship 0.41.3
- Shell type: bash
- Shell version: GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)
- Shell plugin manager: [e.g. oh-my-fish, oh-my-zsh, fisher, antigen]
- Terminal emulator: [e.g. iTerm, Hyper, Terminator]
- Operating system: Linux Mint Debian Edition(LMDE 3 Cindy)

#### Relevant Shell Configuration

My `.bashrc` is too long. Let me know if you need it.

#### Starship Configuration

```bash
# Your configuration here
Don't have any custom configuration yet.
```

**Comment by andytom** [comment]
Hi @gutsytechster, I think this is probably because the font you are using doesn't include these emoji, I think this should just be a case of installing an emoji font e.g the noto emoji font.

**Comment by gutsytechster** [comment]
@andytom Thanks for the quick response. However, I am not sure how can I find the exact font which needs to be installed. 

For the case of `noto` emoji font. I have run the command

```
fc-list | gret "Noto"

```

 to check if I have noto font installed or not.  It seems I do have it installed.
Here is the output of the above command - https://hastebin.com/rudevumowa.makefile`

**Comment by davidkna** [comment]
Debian has a fonts-noto-color-emoji package that should package the emoji font.

**Comment by gutsytechster** [comment]
I have checked all the packages `fonts-noto.*` available as
```
✦ ❯ sudo apt search fonts-noto
i   fonts-noto                              - metapackage to pull in all Noto fonts            
v   fonts-noto:i386                         -                                                  
i   fonts-noto-cjk                          - "No Tofu" font families with large Unicode covera
v   fonts-noto-cjk:i386                     -                                                  
i   fonts-noto-hinted                       - "No Tofu" font families with large Unicode covera
v   fonts-noto-hinted:i386                  -                                                  
i   fonts-noto-mono                         - "No Tofu" monospaced font family with large Unico
v   fonts-noto-mono:i386                    -                                                  
i   fonts-noto-unhinted                     - "No Tofu" font families with large Unicode covera
```
I have checked each of them individually. They seem to be already installed in my system.

**Comment by davidkna** [comment]
You can also get it from [here](https://github.com/googlefonts/noto-emoji/tree/master/fonts). Most other emoji fonts should also work if they are packaged.

**Comment by andytom** [comment]
I've converted this into a discussion rather than a bug since this doesn't seem to be related to starship.
